spec/rubocop/cli/cli_autocorrect_spec.rb

```diff
@@ -517,7 +517,7 @@ def func
             baz
           end
         end
-      RUBY
+    RUBY
```

Comment:
Hm. This looks like the indentation was correct before. It's a continuation from L508, and should be indented one level deeper than that line. Correct?

Reply:
To be honest, I'm not sure. The corrected block aligns the closing heredoc with the equivalent of line 508 which I'd assumed was correct. Is this spec meant to test autocorrection of `InitialIndentation` violations as well as `ClosingHeredocIndentation`?

Reply:
The indentation updates seems reasonable to me. I see we now always align the closing marker with the beginning of the line where the heredoc started.